src/transformer.ts

```diff
@@ -17,7 +17,7 @@ export class EncryptionTransformer implements ValueTransformer {
   }
 
   public to(value?: string | null): string | undefined {
-    if (value === null || value === undefined) {
+    if (!value) {
```

Comment:
This is called when `save` is called on an instance. It will skip empty strings, which should be accepted. Empty strings have encrypted contents that are not null or empty strings.

Reply:
removed the change from 'to', leaved in 'from'